gobblin-utility/src/test/java/org/apache/gobblin/util/PullFileLoaderTest.java

```diff
@@ -218,7 +219,20 @@ public void testRecursiveJobLoadingWithAndGlobalProps() throws Exception {
     Assert.assertEquals(pullFile.entrySet().size(), 7);
   }
 
-
+  @Test
+  public void testJsonPropertyReuseJobLoading() throws Exception {
+    Path path;
+    Config pullFile;
+    path = new Path(this.basePath, "bjob.pull");
+    Config cfg = ConfigFactory.parseMap(ImmutableMap.<String, Object>builder()
+            .put(PullFileLoader.PROPERTY_DELIMITER_PARSING_ENABLED_KEY, true)
+            .build());
+    pullFile = loader.loadPullFile(path, cfg, false);
+    System.out.println(pullFile.getString("json.property.key"));
```

Comment:
remove line 231 and 232

Reply:
I have seen System.out.printlns in many of the Test cases so though it would be okay to keep, I am going to remove it.

Reply:
Disable or enable, I think one is of equivalent possibility in a use case to the other. I saw in your test case that it can help with json object. If that's the motivation, could you encode a json object into a string as a configuration and parse it when used at runtime?

Reply:
I am sorry I could not understand what you meant by the initial statement.
Yes the encoding in configuration and decoding in runtime will work but I wanted to avoid more changes in the runtime code ( i.e decoding of json).

Reply:
My concern is, if the delimiter is disabled for all configurations, some configuration, whose value has the delimiter and expects to be parsed as a list, will break. Is it an issue to you?

Reply:
No it is not going to be an issue for our use case.